docs/easing.md

```diff
@@ -191,7 +191,7 @@ static elastic(bounciness)
 
 A simple elastic interaction, similar to a spring oscillating back and forth.
 
-Default bounciness is 1, which overshoots a little bit once. 0 bounciness doesn't overshoot at all, and bounciness of N > 1 will overshoot about N times.
+Default bounciness is 1, which overshoots a little once. 0 bounciness doesn't overshoot at all, and bounciness of N > 1 will overshoot about N times.
```

Comment:
I don't think the problem here is "bit". The original sentence is lacking a comma before "once". Adding that should help.